tests/sles4sap/qesapdeployment/cluster_add_repos.pm

```diff
@@ -0,0 +1,61 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Deployment steps for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use mmapi 'get_current_job_id';
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+sub run() {
+    my ($self, $run_args) = @_;
+    my $instance = $run_args->{my_instance};
+    record_info("$instance");
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    my $count = 0;
+
+    while (defined(my $maintrepo = shift @repos)) {
+        next if $maintrepo =~ /^\s*$/;
+        qesap_ansible_cmd(cmd => "zypper --no-gpg-checks ar -f -n TEST_$count $maintrepo TEST_$count",
+            provider => $prov);
+        $count++;
+    }
+
+    qesap_ansible_cmd(cmd => "sudo zypper -n ref",
+        provider => $prov);
```

Comment:
I guess L32 & L33 would look better as a single line. I would also use `'sudo zypper -n ref'` instead of `"sudo zypper -n ref"` as there are no variables in the string.

Reply:
Fixed!